.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,18 @@
+name: shellcheck
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt install shellcheck
```

Comment:
Maybe need to ignore the "error" with https://github.com/koalaman/shellcheck/wiki/Ignore?

Reply:
I added a shellcheck disable to the script.